pyflakes/test/test_api.py

```diff
@@ -718,11 +718,7 @@ class IntegrationTests(TestCase):
     Tests of the pyflakes script that actually spawn the script.
     """
 
-    # https://bitbucket.org/pypy/pypy/issues/3069/pypy36-on-windows-incorrect-line-separator
-    if PYPY and sys.version_info >= (3,) and WIN:
-        LINESEP = '\n'
-    else:
-        LINESEP = os.linesep
+    LINESEP = os.linesep
```

Comment:
oh nice! forgot about this -- you can probably inline `os.linesep` now -- no need to have the class variable

Reply:
Done, I'll be back later tonight or tomorrow. Feel free to edit my commits before merging if you need to chane anything.